vue/components/organisms/pickers/pickers.vue

```diff
@@ -307,6 +313,23 @@
     z-index: 1;
 }
 
+.pickers .colors-container .color.disabled {
+    opacity: 0.5;
+    pointer-events: none;
+}
+
+.pickers .colors-wrapper .colors-container .color.disabled:hover > .swatch,
+.pickers .colors-wrapper .colors-container .color.disabled > .swatch {
+    border: 2px solid rgba(128, 128, 128, 0.8);

```

Comment:
I know it's a pain but let's try to use HEXa color format everywhere instead of the RGBa. You can use visual studio code to make the conversion or some online converter if you prefer

Reply:
Nice catch! Went ahead and converted the previous rgba we had as well.